templates/tempest.j2

```diff
@@ -24,6 +24,16 @@ volume_size = 10
 [network-feature-enabled]
 ipv6_subnet_attributes = False
 ipv6 = False
+{% if rally_tempest_floating_ips is defined %}
+floating_ips = {{ rally_tempest_floating_ips }}
+{% endif %}
 
 [identity]
 ca_certificates_file = {{ tempest_ca_path }}
+
+{% if rally_tempest_extra_settings is defined %}
+# Extra tempest.conf settings
+{% for param in rally_tempest_extra_settings %}
```

Comment:
Is there a guarantee that all the settings will be added in the same order as they are defined?

Reply:
Also this feels a bit hackerish.

Reply:
Yes. I don't know what that is. The data I think is in YAML lists which are different than python lists?